opentelemetry/proto/logs/v1/logs.proto

```diff
@@ -27,7 +27,7 @@ option go_package = "github.com/open-telemetry/opentelemetry-proto/gen/go/logs/v
 // A collection of InstrumentationLibraryLogs from a Resource.
 message ResourceLogs {
   // The resource for the logs in this message.
-  // If this field is not set then no resource info is known.
+  // If this field is not set then no resource info is unknown.
```

Comment:
Too many negations here: "If this field is **not** set then **no** resource info is **un**known.

Reply:
good catch
```suggestion
  // If this field is not set then resource info is unknown.
```